src/client/src/rt-platforms/glue/glue.ts

```diff
@@ -0,0 +1,96 @@
+import Glue, { Glue42 as GlueInterface } from '@glue42/desktop'
+import Glue4Office, { Glue42Office as Glue42OfficeInterface } from '@glue42/office'
+import { CurrencyPairPositionWithPrice } from 'rt-types'
+import { WindowConfig } from '../types'
+import { BasePlatformAdapter } from '../platformAdapter'
+import { openGlueWindow, registerWindowMethods } from './window'
+import { sendNotification, NotifyPermission } from './utils/sendNotification'
+import { getAddinStatus, openSheet, updateSheet, BlotterData } from './utils/excel'
+import DefaultRoute from 'rt-platforms/defaultRoute'
+import { ApplicationEpic } from 'StoreTypes'
+import { GlueHeader, GlueLogoLink } from './'
+
+// TODO remove onGlueLoaded and its usages when the whole app is rendered after glue has loaded
+export const onGlueLoaded = (callback: () => void) => {
+  const interval = setInterval(() => {
```

Comment:
There is a `waitForObject` util you can use instead (which does the same thing)

Reply:
It seems to have a slightly different signature, which makes it a bit more complex to use in our case inside the useEffect() hook, but this check would be removed when the app is loaded asynchronously inside the Glue().then() anyway. However, we have slightly refactored this and temporarily created a custom hook called useOnGlueLoaded